Gir.toml

```diff
@@ -283,6 +295,7 @@ manual = [
     "Gtk.Allocation",
     "Gtk.AppChooser",
     "Gtk.EntryBuffer",
+    "Gtk.Plug",
```

Comment:
This seems only for xlib, so needed some cfg_condition = "unix" or still be ignored

Reply:
Do we have implemented a cfg parameter in gir or not yet?

Reply:
And it works on OSX, so I suppose that the current condition is fine?

Reply:
Yes we already implement it.